utils/numcheck/calcderivs

```diff
@@ -168,18 +182,18 @@ def calculate_reference(binary, reffile, coords, specienames, species, origin,
 
 
 def calculate_forces(binary, disp, coords, specienames, species, origin,
-                     latvecs, chargeRestart):
+                     latvecs, chargeRestart, direction, atoms):
```

Comment:
I think we should stick to PEP 8 for naming variables (avoid mixedCase), not our Fortran conventions (lowerCamelCase):
https://peps.python.org/pep-0008/#function-and-variable-names